### ISSUE TITLE
Add support for object categories

### ISSUE THREAD

**Issue by jpsember** [jpsember/java-ml#59]
I haven't incorporated this into the yolo loss function yet.

**Comment by jpsember** [comment]
Relevant:  https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

**Comment by jpsember** [comment]
- [x] Is there some sort of normalization I should apply to the crossentropy loss?

**Comment by jpsember** [comment]
https://stackoverflow.com/questions/49390842/cross-entropy-in-pytorch

> PyTorch's CrossEntropyLoss expects unbounded scores (interpretable as logits / log-odds) as input, not probabilities (as the CE is traditionally defined).
